package.json

```diff
@@ -9,18 +9,20 @@
     "src"
   ],
   "scripts": {
+    "#postinstall": "postinstall hook is required to 'fix' ava d.ts file",
+    "postinstall": "rimraf node_modules/ava/index.d.ts",
```

Comment:
The ava typings contain references to the promise object which is only available in typescript with compile to es2015.
paeckchen currently compiles to es5. Therefore the typings need to be patched.